include/fiction/algorithms/simulation/sidb/convert_potential_to_distance.hpp

```diff
@@ -0,0 +1,57 @@
+//
+// Created by Jan Drewniok on 10.11.23.
+//
+
+#ifndef FICTION_CONVERT_POTENTIAL_TO_DISTANCE_HPP
+#define FICTION_CONVERT_POTENTIAL_TO_DISTANCE_HPP
+
+#include "fiction/algorithms/simulation/sidb/sidb_simulation_parameters.hpp"
+#include "fiction/technology/physical_constants.hpp"
+
+#include <cmath>
+#include <cstdint>
+#include <limits>
+#include <vector>
+
+namespace fiction
+{
+/**
+ * This function converts a given electrostatic potential to the corresponding distance for given
+ * physical parameters via the screened Coulomb potential.
+ *
+ * @param params The simulation parameters.
+ * @param potential The electrostatic potential (unit: V) to be converted to a distance (unit: nm).
+ * @param precision The precision level for the conversion, specifying the number of decimal places.
+ * @return The distance (unit: nm) corresponding to the given electrostatic potential (unit: V).
+ */
+[[nodiscard]] double convert_potential_to_distance(const sidb_simulation_parameters& params, const double potential,
```

Comment:
In my opinion, `potential` should go first, then `params` (defaulted), then `precision` (defaulted, if necessary at all)